docs/sample-solutions/windows-server/healthcare-multitier-app.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Windows Server on Azure healthcare front-end application
+description: Proven solution for building a secure, multi-tier web application with Windows Server on Azure that uses scale sets, Application Gateway, and load balancers.
+author: iainfoulds
+ms.date: 06/26/2018
+---
+# Deploy a secure and scalable multi-tier web application on Windows Server for a healthcare solution
+
+This sample solution is applicable to healthcare organizations that have a need to secure multi-tier applications. In this solution, a front-end ASP.NET application securely connects to a protected back-end Microsoft SQL Server cluster.
+
+Example application scenarios include running operating room applications, patient appointments and records keeping, or prescription refills and ordering. Traditionally, organizations had to maintain legacy on-premises applications and services for these scenarios. With a secure way and scalable way to deploy these Windows Server applications in Azure, organizations can modernize their deployments are reduce their on-premises operating costs and management overhead.
+
+## Potential use cases
+
+You should consider this solution for the following use cases:
+
+* Modernizing application deployments in a secure cloud environment.
+* Reducing legacy on-premises application and service management.
+* Improving patient healthcare and experience with new application platforms.
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
+
+![Architecture overview of the Azure components involved in multi-tier Windows Server application for healthcare][architecture]
+
+## Architecture
+
+This solution covers a multi-tier healthcare application that uses ASP.NET and Microsoft SQL Server. The data flows through the solution as follows:
+
+1. Users access the front-end ASP.NET healthcare application through an Azure Application Gateway.
+2. The Application Gateway distributes traffic to VM instances within an Azure virtual machine scale set.
+3. The ASP.NET application connects to Microsoft SQL Server cluster in a back-end tier via an Azure load balancer. These backend SQL Server instances are in a separate Azure virtual network, secured by network security group rules that limit traffic flow.
+4. The load balancer distributes SQL Server traffic to VM instances in another virtual machine scale set.
```

Comment:
Is this using Always On Availability Groups? Is that compatible with scale sets? If you scale out, how would they join the cluster?

Reply:
I believe so, yes. Could use PowerShell DSC or Custom Script Extension to configure instances. Could switch to just VMs, but then would lose any scaling benefits, and internal load configuration isn't automatic.

Reply:
Is there a benefit to scaling out the availability group? Writes will always go to the primary replica.

Reply:
If you had a business desire to increase the secondary replicas, you would just need to manually scale out the scale set to add another instance and wire it into the load balancer automatically. I don't have autoscale rules implemented as I didn't see the value in it since it doesn't distribute load for performance reasons like you'd typically choose to scale out. As the instances are distributed across Availability Zones, the scale set should also be able to respond to an outage and deploy additional instances, which wouldn't be the case in a straight VM deployment.

Again, if you'd prefer, can just switch this to straight VMs.